ts/Extensions/OverlappingDataLabels.ts

```diff
@@ -190,7 +190,7 @@ Chart.prototype.hideOverlappingLabels = function (
                     left: '0',
                     center: '0.5',
                     right: '1'
-                }[label.alignValue as Highcharts.AlignValue];
+                }[label.alignValue as Highcharts.AlignValue || label.textAlign];
```

Comment:
I might be wrong here, but isn't `align` and `textAlign` different options? At least that's the case in StackLabels: https://api.highcharts.com/highcharts/yAxis.stackLabels.textAlign

Reply:
It seems that in case when the `useHTML` is set to true, the `textAlign` property is created in the `alignSetter` for HTML elements (https://github.com/highcharts/highcharts/blob/vacation2020/js/Core/Renderer/HTML/HTML.js#L289) and it receives the value of the `align` option. I suppose it works that way to ensure the right position of a text for elements (e.g. https://api.highcharts.com/highcharts/title) that don't provide `textAlign` as an option that can be set through the API.